packages/orbit-components/src/Modal/index.tsx

```diff
@@ -440,15 +440,13 @@ const Modal = React.forwardRef<Instance, Props>(
               "lm:relative lm:bottom-auto lm:pb-0 lm:overflow-visible",
               "lm:[&_.orbit-modal-section:last-of-type]:pb-xxl lm:[&_.orbit-modal-section:last-of-type:after]:content-none lm:[&_.orbit-modal-section:last-of-type]:mb-[var(--orbit-modal-footer-height,0px)]",
               "lm:[&_.orbit-modal-mobile-header]:w-[calc(var(--orbit-modal-width)-48px-theme(spacing.xxl))]",
+              "lm:rounded-modal",
```

Comment:
I am now wondering why we have this and not `lm:rounded-modal-mobile` 🤔 Figma only seems to have one… maybe we should check with designers

Reply:
I could find this in Figma but I'm not sure I'm looking at the right place.

**Desktop library**

Corner radius : 9px

https://www.figma.com/file/nkZ37lByVmAGOqim5PhbZT/%5BOrbit-Design-System%5D-Desktop-Web-Components-(Community)?type=design&node-id=2886-5088&mode=design&t=yYpucR259fCAlBqO-0

**Responsive library**

Corner radius (only top): 12px

https://www.figma.com/file/eDQ3YAPCK9K6bpGxFsu6J8/%5BOrbit%5D-Responsive-Web-Components-(Community)?type=design&node-id=3729-57&mode=design&t=hU6SKQzdiNdS4T3i-0

Reply:
To increase the entropy, I saw this as the desktop library:
https://www.figma.com/file/Y79I18H9w8HJZVxPdAkAHb/%5BOrbit%5D-Desktop-Components?type=design&node-id=7757-12757&mode=design&t=jTpTgtdVycsPmmeR-0

Reply:
After consulting with the Orbit designers, we confirmed that all the rounded corners must be 12px.